src/cplscheme/config/CouplingSchemeConfiguration.cpp

```diff
@@ -599,11 +591,24 @@ void CouplingSchemeConfiguration::addBaseAttributesTagConvergenceMeasure(
   tag.addAttribute(attrStrict);
 }
 
+void CouplingSchemeConfiguration::addTagMinIterations(
+    xml::XMLTag &tag)
+{
+  using namespace xml;
+  XMLTag tagMinIterations(*this, TAG_MIN_ITERATIONS, XMLTag::OCCUR_NOT_OR_ONCE);
+  tagMinIterations.setDocumentation("Allows to specify a minimum amount of iterations per time window at from which point convergence measures are considered. "
+                                    "Note that not defining convergence measures is interpreted as \"no convergence\".");
```

Comment:
```suggestion
  tagMinIterations.setDocumentation("Allows to specify a minimum amount of iterations that must be performed per time window.");
```

Reply:
I think mixing what `min-iterations` and convergence measures here is not really necessary and overly complicated. If I do not define a convergence measure then I should also not have to worry about it, no?

The information how convergence measures and min/max iterations interact should come somewhere else.